planner/core/src/main/java/eu/seaclouds/platform/planner/core/HttpHelper.java

```diff
@@ -83,22 +84,48 @@ public String getRequest(String restPath, List<NameValuePair> params){
         }
     }
 
+    public String postInBody(String restPath, String bodyContent){
+        HttpPost post = new HttpPost(prepareRequestURL(restPath));
+        try {
+            StringEntity entity = new StringEntity(bodyContent);
+            post.setEntity(entity);
+            HttpResponse response = httpclient.execute(post);
+            String result = EntityUtils.toString(response.getEntity());
+            return result;
+        }catch (ClientProtocolException e) {
+            return null;
```

Comment:
That is the next step. Adding uniform error handling. I was looking for some guidelines of SeaClouds error handling policies. Where can I find one?